wire-ios-request-strategy/Sources/Message Sending/MessageSender.swift

```diff
@@ -395,6 +395,8 @@ public final class MessageSender: MessageSenderInterface {
             )
 
         await context.perform {
+            // handle 201 case failed_to_send
```

Comment:
question: is this todo or it's done inside the next line?

Reply:
it's not a todo, it's a comment of the next line